doc/developers/utilities.rst

```diff
@@ -43,6 +43,11 @@ should be used when applicable.
   be sliced or indexed using safe_index.  This is used to validate input for
   cross-validation.
 
+Some estimators are able to store intermediate results using ``Memory``
+instance. The func:`check_memory`: check that memory is a valid input.
```

Comment:
problem with the rst syntax (missing colon at the beginning) this should be 
```
:func:`check_memory`
```

Please check other potential problems in the doc generated by CircleCI on your PR: https://13139-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/developers/utilities.html